### DISCUSSION TITLE
Share your experience on Postman Flows

### DISCUSSION THREAD

**Discussion by anudeepmedicharla** [postmanlabs/postman-flows#308] [Seeking Feedback]
Hi everyone! 

We created a short form to help us understand how you feel about different parts of the Flows product. This will help us understand where to focus our efforts on. We'd appreciate a lot if you can take just a couple of minutes and give your thoughts via the form below :) 

https://forms.gle/aojH9Jy7nXx8LRFm8

cc: @samay-v @saswatds

**Comment by AL1nga** [comment]
Hi,

I have a flow that sends request to an API which requires BasicAuth. When I run the flow for the first time the Send Request block fails. But when I run it again it was successful. Any idea how I can fix this behavior?
